src/components/Sum.jsx

```diff
@@ -1 +1,16 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => (
```

Comment:
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```

I'm not sure, but maybe with destructuring this syntax will be better than defaultProps